src/components/molecules/collabsible-tree/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React from "react"
+import clsx from "clsx"
+
+import MoreHorizontalIcon from "../../fundamentals/icons/more-horizontal-icon"
+import MinusIcon from "../../fundamentals/icons/minus-icon"
+import PlusIcon from "../../fundamentals/icons/plus-icon"
+import Button from "../../fundamentals/button"
+import Actionables, { ActionType } from "../actionables"
+
+type CollabsibleTreeProps = {
```

Comment:
small typo:

```diff
- Collabsible
+ Collapsible
```

Applies for all occurrences + folder name but will only mention it here 😄